tests/quantization/test_quantized_module.py

```diff
@@ -351,6 +351,110 @@ def test_quantized_module_rounding_fhe(model_class, input_shape, default_configu
     # TODO: https://github.com/zama-ai/concrete-ml-internal/issues/3800
 
 
+@pytest.mark.parametrize("model_class, input_shape", [pytest.param(FC, (100, 32 * 32 * 3))])
+def test_statuses(model_class, input_shape, default_configuration):
+    """Check that giving statuses work properly."""
+
+    torch_fc_model = model_class(activation_function=nn.ReLU)
+    torch_fc_model.eval()
+
+    # Create random input
+    numpy_input = numpy.random.uniform(size=input_shape)
+    torch_input = torch.from_numpy(numpy_input).float()
+    assert isinstance(torch_input, torch.Tensor)
+    assert isinstance(numpy_input, numpy.ndarray)
+
+    # Compile with rounding activated
+    with pytest.raises(ValueError):
```

Comment:
I would prefer `with pytest.raises(ValueError, match=...)` here, especially since we have two different types of value errors, it's more robust !

Reply:
Indeed, I was a bit lazy there!
Will fix this